samples/client/petstore/csharp/OpenAPIClient/src/Org.OpenAPITools/Model/EnumTest.cs

```diff
@@ -122,18 +120,16 @@ public enum EnumIntegerEnum
         [JsonConverter(typeof(StringEnumConverter))]
         public enum EnumNumberEnum
         {
-            
             /// <summary>
             /// Enum NUMBER_1_DOT_1 for value: 1.1
             /// </summary>
-            [EnumMember(Value = "1.1")]
-            NUMBER_1_DOT_1 = 1,
-            
+            NUMBER_1_DOT_1 = 1.1,
+
             /// <summary>
             /// Enum NUMBER_MINUS_1_DOT_2 for value: -1.2
             /// </summary>
-            [EnumMember(Value = "-1.2")]
-            NUMBER_MINUS_1_DOT_2 = 2
+            NUMBER_MINUS_1_DOT_2 = -1.2
```

Comment:
@jimschubert @albator1932 For double/float value, do we need to restore the old behavior (using the string representation of the value)?

Reply:
Looks like it's not ok:
```
  Model\EnumTest.cs(126,30): error CS0266: Cannot implicitly convert type 'double' to 'int'. An explicit conversion exists (are you missing a cast?) [C:\projects\openapi-generator\samples\client\petstore\csharp\OpenAPIClient\src\Org.OpenAPITools\Org.OpenAPITools.csproj]
  Model\EnumTest.cs(131,36): error CS0266: Cannot implicitly convert type 'double' to 'int'. An explicit conversion exists (are you missing a cast?) [C:\projects\openapi-generator\samples\client\petstore\csharp\OpenAPIClient\src\Org.OpenAPITools\Org.OpenAPITools.csproj]
```
Let me fix it.

Reply:
Fixed via https://github.com/OpenAPITools/openapi-generator/pull/774/commits/ec5ca857b39bf4692b1d14107e0db731a84bea29